test/units/TEST-65-ANALYZE.sh

```diff
@@ -990,6 +990,9 @@ systemd-analyze architectures uname
 systemd-analyze smbios11
 systemd-analyze smbios11 -q
 
+systemd-analyze chid ||:
+systemd-analyze chid --json=pretty ||:
```

Comment:
I do not think we should ignore errors.
Maybe better to conditionalize with `detect-virt -c`. Also, please parse the json output with `jq`.